binder/README

```diff
@@ -11,3 +11,5 @@ Please note that this setup is developer focused.
 For a more user-focused Binder use this URL:
 
 https://mybinder.org/v2/gh/jupyterlab/jupyterlab-demo/master?urlpath=lab/tree/demo/Lorenz.ipynb
+
+If there is an error with the launched application, you can browse to the ` `/edit/jupyterlab-dev.log` url to see the server logs.
```

Comment:
That's very helpful information! I was wondering how to get to that log!

Please note, however, that this is not a Markdown file, therefore the backticks have no effect (except displaying backticks).

You probably want to add a suffix `.md`?

And even if you do, I think there is one backtick too many?

Finally, you could also make it nicer for people reading this sentence in a non-breaking and non-wide environment and graciously add a line break.

Reply:
@blink1073 Another problem with your edit is that your sentence comes after the "user-focused" URL, but it is really meant as note about the "developer-focused" URL a few lines above, isn't it?